docs/source/intro/get_started.rst

```diff
@@ -4,11 +4,11 @@
 Getting Started
 ****************
 
-A working AiiDA installation consists of three core components, plus any external codes you wish to run:
+An AiiDA installation consists of three core components (plus any external codes you wish to run):
 
-* aiida-core: The main Python package and the associated command line tool: ``verdi``.
-* |PostgreSQL|: A service which manages the database that AiiDA uses to store generated data.
-* |RabbitMQ|: A service which manages communication with the processes that AiiDA spawns.
+* ``aiida-core``: The main Python package and the associated ``verdi`` command line interface
```

Comment:
Why would we typeset `aiida-core` as code, when it just represents a name in this context like all the other components?

Reply:
I think you're right, I'll revert that